cookbooks/convection-box.prm

```diff
@@ -101,7 +101,9 @@ end
 
 subsection Formulation
   set Formulation = custom
-  set Temperature density approximation = adiabatic
+  set Mass conservation    = incompressible
```

Comment:
or should we just remove the whole ``Formulation`` block and go with the default for now?